docs/guides/serverless-rest-api-example.md

```diff
@@ -391,3 +394,369 @@ curl --location --request PUT '{url}' \
 ```bash
 curl --location --request GET 'http://localhost:9001/apis/public/profiles/{id}/image/download'
 ```
+
+{% /tab %}
+{% tab label="Python" %}
+
+## What we'll be doing
+
+1. Use Nitric to create an API to create and update profiles
+2. Create handlers for the following API operations
+
+| **Method** | **Route**      | **Description**                  |
+| ---------- | -------------- | -------------------------------- |
+| `GET`      | /profiles/{id} | Get a specific profile by its Id |
+| `GET`      | /profiles      | List all profiles                |
+| `POST`     | /profiles      | Create a new profile             |
+| `DELETE`   | /profiles/{id} | Delete a profile                 |
+| `PUT`      | /profiles/{id} | Update a profile                 |
+
+3. Run locally for testing
+4. Deploy to a cloud of your choice
+5. (Optional) Add handlers for the following API operations
+
+| **Method** | **Route**                    | **Description**                  |
+| ---------- | ---------------------------- | -------------------------------- |
+| `GET`      | /profiles/{id}/image/upload  | Get a profile image upload URL   |
+| `GET`      | profiles/{id}/image/download | Get a profile image download URL |
+
+## Prerequisites
+
+- [Pipenv](https://pypi.org/project/pipenv/) - for simplified dependency management
+- The [Nitric CLI](https://nitric.io/docs/installation)
+- An [AWS](https://aws.amazon.com), [GCP](https://cloud.google.com) or [Azure](https://azure.microsoft.com) account (_your choice_)
+
+## Getting started
+
+We’ll start by creating a new project for our API.
+
+```bash
+nitric new
+```
+
+Create a project name, select the TypeScript template and choose the default glob handler.
+
+```bash
+? What is the name of the stack? my-profile-api
+? Choose a template: official/Python - Starter
+? Glob for the function handlers? functions/*.py
+```
+
+Next, open the project in your editor of choice.
+
+```bash
+> cd my-profile-api
+```
+
+Make sure all dependencies are resolved with pipenv.
+
+```bash
+pipenv install
+```
+
+The scaffolded project should have the following structure:
+
+```text
++--functions/
+|  +-- hello.py
++--nitric.yaml
++--Pipfile
++--Pipfile.lock
++--README.md
+```
+
+You can test the project scaffold with the following command.
+
+```bash
+pipenv run dev
+```
+
+> _Note:_ the script in the template starts the Nitric Server using `nitric start` and runs your functions.
+
+> Since we won't use the example function you can now delete the `functions/hello.py` file.
+
+## Coding our Profile API
+
+Let's start by initializing our profiles api, create a file named 'profiles.py' within functions and add the following code.
+
+```python
+from uuid import uuid4
+
+from nitric.resources import api, collection, bucket
+from nitric.application import Nitric
+from nitric.api.storage import FileMode
```

Comment:
This import isn't needed, we can use the `file.upload_url()` and `file.download_url()` methods instead